README.md

```diff
@@ -1 +1,43 @@
-# first-capstone
\ No newline at end of file
+# capstone project
+
```

Comment:
- [ ] [Optional] I strongly suggest you add a brief description for your project